### ISSUE TITLE
Document dropbox sharing 

### ISSUE THREAD

**Issue by eddy-geek** [opoto/wtracks#33]
Documenting how to share a link to a gpx hosted on Dropbox

Sample link when sharing in Dropbox: `https://www.dropbox.com/s/z2fliillu16nzc9/grand-paradis_raid_5j.gpx?dl=0`

First one has to change to `dl=1` at the end as explained [by dropbox](https://help.dropbox.com/files-folders/share/force-download))
This does not seem to be enough for WTracks, maybe it can't follow double-redirect 301 + 302

![image](https://user-images.githubusercontent.com/2772505/147420876-d54ff95a-800f-4115-be88-00371de9074e.png)

What does work is: `https://www.dropbox.com/s/dl/z2fliillu16nzc9/grand-paradis_raid_5j.gpx` → [Full WTracks URL](https://opoto.github.io/wtracks/?url=https://www.dropbox.com/s/dl/z2fliillu16nzc9/grand-paradis_raid_5j.gpx)

Not sure if you want this in the docs, otherwise feel free to close, it's public now :smile:

**Comment by opoto** [comment]
Yes, this can be documented in https://opoto.github.io/wtracks/doc. It can be added at the end of the "sharing". Hoping that someone will find it here...
I can add it when I'm back from vacation.
Thanks
